src-built-in/components/apitest/test1.js

```diff
@@ -0,0 +1,1538 @@
+//setup all the clients to be used
+const dragdropClient = FSBL.Clients.DragAndDropClient;
+const hotkeyClient = FSBL.Clients.HotkeyClient;
+const distributedStoreClient = FSBL.Clients.DistributedStoreClient
+const launcherClient = FSBL.Clients.LauncherClient
+const windowClient = FSBL.Clients.WindowClient
+const linkerClient = FSBL.Clients.LinkerClient
+const loggerClient = FSBL.Clients.Logger
+const routerClient = FSBL.Clients.RouterClient
+const workspaceClient = FSBL.Clients.WorkspaceClient
+const storageClient = FSBL.Clients.StorageClient
+const dialogManager = FSBL.Clients.DialogManager
+const searchClient = FSBL.Clients.SearchClient
+
+const keyMap = FSBL.Clients.HotkeyClient.keyMap;
+var store, test2SpawnResopnse
+
+function openTab(evt, tabName) {
+	var i, tabcontent, tablinks;
+	tabcontent = document.getElementsByClassName("tabcontent");
+	for (i = 0; i < tabcontent.length; i++) {
+		tabcontent[i].style.display = "none";
+	}
+	tablinks = document.getElementsByClassName("tablinks");
+	for (i = 0; i < tablinks.length; i++) {
+		tablinks[i].className = tablinks[i].className.replace(" active", "");
+	}
+	document.getElementById(tabName).style.display = "block";
+	evt.currentTarget.className += " active";
+}
+
+function renderPage() {
+	/* Setup dragevent listener for inputbox */
+	document.getElementById('symbolInput').addEventListener('dragstart', function (event) {
+		var data = {
+			'rsrchx.report': {
+				symbol: event.target.value
+			},
+			'symbol': event.target.value
+		};
+		dragdropClient.dragStartWithData(event, data);
+	})
+
+	routerClient.onReady(function () {
+		console.log('RouterClient is ready')
```

Comment:
In general, I think it would be better to have the output available from the UI rather than having to open the dev console.